docs/book/features/block_volume.md

```diff
@@ -375,3 +376,67 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Cluster<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in a Tanzu Kubernetes Grid (TKG) Cluster using an unused PersistentVolumeClaim (PVC) from the Project Pacific cluster only if the latter satisfies the following conditions:
+
+- It is present in the same namespace as the TKG Cluster
+
+- It is associated with the same `StorageClass` as the default storage class in the TKG Cluster
```

Comment:
Are you sure that this is a requirement? AFAIK storage class doesnt need to be added to spec for static provisioning.

Reply:
Yes, changing this. Balu and Sandeep pointed this out too.

Reply:
There are no default storage classes in TKG AFAIK.
This is from my TKG cluster
```
root@4206260419b17c5559f7d5ab02249adc [ ~ ]# kubectl get sc -o yaml
apiVersion: v1
items:
- apiVersion: storage.k8s.io/v1
  kind: StorageClass
  metadata:
    creationTimestamp: "2020-08-07T09:02:36Z"
    labels:
      isSyncedFromSupervisor: "yes"
    name: gc-storage-profile
    resourceVersion: "431"
    selfLink: /apis/storage.k8s.io/v1/storageclasses/gc-storage-profile
    uid: 7d8e08a4-7735-4cb2-b477-bfe7758ac4c9
  parameters:
    svStorageClass: gc-storage-profile
  provisioner: csi.vsphere.vmware.com
  reclaimPolicy: Delete
  volumeBindingMode: Immediate
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
root@4206260419b17c5559f7d5ab02249adc [ ~ ]#
```
if there is no default SC then PV will not get assigned to it and if it not tied to a SC with `allowexpansion` set to true we won't be able to resize it

Reply:
Changed it.